tools/pv_tree_generator/pop_obs_spec_common.textproto

```diff
@@ -2024,4 +2078,83 @@ spec: {
     mdenom: "PerCapita"
     name: "Purchasing Power Parity Per Capita"
   }
+}
+
+spec:{
+  pop_type: "Legislation"
+  cprop: "policyType"
+  dpv: {
+    prop: "category"
+    val: "COVID19Pandemic"
+  }
+  dpv: {
+    prop: "legislationPassedBy"
+    val: "GovernmentOrganization"
+  }
+  obs_props: {
+    mprop: "policyExtent"
+  }
+  name: "COVID-19 Policy"
+  vertical: "Health"
```

Comment:
@beets @tjann any suggestion on the vertical for this observation?

Reply:
Health seems good. One alternative is to make COVID it's own vertical.

Reply:
Can we sort the values under Health?

Reply:
And make plural/singular case consistent?

Reply:
Pulling covid out into a new category sgtm

Reply:
Added a new category and always use plural cases